backend/test/capsule.unit.test.ts

```diff
@@ -0,0 +1,71 @@
+import { BigNumber, Contract, ContractFactory } from "ethers";
+import { expect, assert } from "chai";
+const { ethers, network } = require("hardhat");
+
+describe("capsule unit tests", function () {
+    const amount1 = ethers.utils.parseEther("1000");
```

Comment:
//you need to do
let capsuleFactory: CapsuleFactory,
        erc20Capsule: Erc20Capsule,
  //   etc......

//also
        user: signer,
        user2: signer;